### PR TITLE
feature/admin_message_history

### PR DESCRIPTION
**Pull request by Rodrigo-Miranda-0**
#### Board:
* [Ticket #VKKVq7Lc](https://trello.com/c/VKKVq7Lc/31-3-as-an-admin-i-should-be-able-to-see-the-message-history-in-each-conversation)
---
#### Description:

* Allows the admin to view the messages corresponding to each conversation
---
#### Notes:
*
---
#### Tasks:
- [x] ADMIN: View messages from certain conversations
---
#### Risk:
*
---
#### Preview:

### CODE REVIEW

File: app/admin/conversation.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ActiveAdmin.register Conversation do
+  index do
+    selectable_column
+    id_column
+    column :user1
+    column :user2
+    column "Messages" do |conversation|
+      link_to "View Messages", admin_messages_path(q: { conversation_id_eq: conversation.id }), label: "Messages"
+    end
```
Comment (jpascual1994) [inline app/admin/conversation.rb]: I think that the idea of this card is to display the message inside of the conversation page (#show)

Comment (juanmferreira93) [reply app/admin/conversation.rb]: yep, for the user, not for the admin (also here could be ok, but need to be on the user side)

**Review by jpascual1994 — COMMENTED** [review]
looks good but there are some changes on the schema that shouldn't be present on this PR

**Review by jpascual1994 — APPROVED** [review]

**Review by juanmferreira93 — APPROVED** [review]

**Review by lauperalti — APPROVED** [review]
🚀

**Pull request merged**